### ISSUE TITLE
query value array types 

### ISSUE THREAD

**Issue by haavardj** [jepiqueau/capacitor-sqlite#31]
There are some inconsistencies in how the values[] array type is handled between iOS and Android.

On iOS queries, values array _must_ be string[]. In Plugin.swift line 204
```
        guard let values = call.options["values"] as? Array<String> else {
            retValues(call:call,ret:[],message:"Query command failed : Values should be an Array of string")
            return
        }
````

On Android, any[] is accepted and the plugin will seemingly happily convert to strings. In CapacitorSQLite.java line 229
```
            ArrayList<String> vals = new ArrayList<String>();
            for (int i = 0; i < values.length(); i++) {
                vals.add(values.getString(i));
            }
```

This feature implies that something that works perfectly on Android may fail on iOS. This is unfortunate.

**Comment by jepiqueau** [comment]
@haavardj Thanks for using the plugin and pointing me out some inconsistencies. The Android plugin has been fixed in release 2.2.1-3. It must be an Array<String>. When you have test it please close the issue

**Comment by haavardj** [comment]
It now correctly throws 'Must provide an Array of strings' on Android. Thanks!
